content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -99,7 +99,7 @@ different Kubernetes components.
 | `DownwardAPIHugePages` | `true` | Beta | 1.22 | |
 | `EndpointSliceTerminatingCondition` | `false` | Alpha | 1.20 | 1.21 |
 | `EndpointSliceTerminatingCondition` | `true` | Beta | 1.22 | |
-| `ExpandedDNSConfig` | `false` | Alpha | 1.22 | |
+| `ExpandedDNSConfig` | `true` | Beta | 1.26 | |
```

Comment:
Rather than **replacing** the existing line, we are supposed to **add a new line** for the Beta stage. The `GracefulNodeShutdown` feature can be used as a reference.

Reply:
thanks, done.